### ISSUE TITLE
iOS build instructions

### ISSUE THREAD

**Issue by weepy** [keharriso/love-nuklear#36]
hiya - do you know if it's possible to build this for iOS ?

**Comment by keharriso** [comment]
Yes, it's definitely possible, but I don't have the hardware to test it. Additionally, you'd have to build LÖVE from the source code. The process should be something like this:
* Download LÖVE libraries and source code for iOS
* Compile LÖVE for iOS
* Run CMake and set LUA_INCLUDE_DIR to be the include directory for luajit and LUA_LIBRARY to be the luajit shared library
* Generate Makefiles and run `make`
